README.md

```diff
@@ -1,3 +1,9 @@
+This repo depends on our [fork of ichord/At.js](https://github.com/CenterForOpenScience/At.js) which
+contains a bug fix for @ mentions. If the PR [ichord/At.js#394](https://github.com/ichord/At.js/pull/394)
+is merged then the original repo will be used instead of our fork.
```

Comment:
Sorry I wasn't clear: I was thinking we would add a notice to the README in the fork, not in the OSF repo. My reasoning is that someone would discover https://github.com/CenterForOpenScience/At.js in [`bower.json`](https://github.com/CenterForOpenScience/osf.io/blob/0c51337fa1034dbeac751a98da63b3bc8492fc06/bower.json#L23) and visit the URL—we should give them a pointer from there to the upstream project.